src/nwm_routing/src/nwm_routing/output.py

```diff
@@ -91,6 +91,10 @@ def nwm_output_generator(
             [pd.DataFrame(r[1], index=r[0], columns=qvd_columns) for r in results],
             copy=False,
         )
+        
+        # todo: create a unit test by saving FVD array to disk and then checking that
+        # it matches FVD array from parent branch or other configurations. 
+        # flowveldepth.to_pickle(output_parameters['test_output'])
```

Comment:
For future effort. Simply saving the FVD array to disk as pickle facilitates quick testing.

Reply:
Pickle is fast but has inherent downsides. It would be worth a very quick effort to test numpy.save or one of the other methods mentioned in the replies [here](https://stackoverflow.com/questions/9619199/best-way-to-preserve-numpy-arrays-on-disk).